src/jwplayer/players/DesktopArticleVideoPlayer/DesktopArticleVideoPlayer.tsx

```diff
@@ -50,23 +58,31 @@ export const DesktopArticleVideoPlayerContent: React.FC<DesktopArticleVideoPlaye
 							isScrollPlayer ? styles.desktopArticleVideoWrapperScrollPlayer : styles.desktopArticleVideoWrapper
 						}
 					>
-						{adComplete && (
+						{adEngineComplete && (
 							<div>
 								<div className={styles.topBar}>
 									{!isScrollPlayer && <UnmuteButton />}
 									{isScrollPlayer && (
 										<CloseButton dismiss={() => setDismissed(true)} iconColor={'#fff'} className={styles.closeButton} />
 									)}
 								</div>
-								<JwPlayerWrapper
-									getDismissed={getDismissed}
-									config={getArticleVideoConfig(videoDetails)}
-									onReady={(playerInstance) => {
-										articlePlayerOnReady(videoDetails, playerInstance);
-										setIsPlayerReady(true);
-									}}
-									stopAutoAdvanceOnExitViewport={false}
-								/>
+								{jwpAdsSetupComplete.strategyRulesEnabled && <div className="strategyRulesWrapper" />}
```

Comment:
IMO it should be somewhere inside `<JwPlayerWrapperWithStrategyRules>` component or we should create another wrapping component that will contain this div and `<JwPlayerWrapperWithStrategyRules>`